net-dns/dnsdist/files/dnsdist.service

```diff
@@ -0,0 +1,24 @@
+[Unit]
+Description=dnsdist
+Wants=network-online.target
+After=network-online.target
+
+[Service]
+# Note: when editing the ExecStart command, keep --supervised and --disable-syslog
+ExecStart=/usr/bin/dnsdist --supervised --disable-syslog -u dnsdist -g dnsdist
+
+Type=notify
```

Comment:
Upstream says:

> Should dnsdist be run on a system with systemd, it is highly recommended to have the systemd header files (libsystemd-dev on Debian and systemd-devel on CentOS) installed to have dnsdist support systemd-notify.

Therefore I am a bit confused.
It will work without, however without the systemd-notify support.